### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by Antosik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,47 @@
...
+
+## Как помочь разработке
+
+- [Делитесь с нами](https://github.com/Antosik/lol-guilds-client/issues) вашими идеями
+- Если нашли баг - [напишите](https://github.com/Antosik/lol-guilds-client/issues) об этом
+
+
+## Ссылки
+
+- [Благодарности](https://github.com/Antosik/lol-guilds-client/wiki/Credits)
```
Comment (kozhin) [inline README.md]: Эндрю у нас Rainfeather )

File: README.md [hunk 2]
```diff
@@ -1,17 +1,47 @@
...
 
-## How to install
-Download and run an installer that related to your operating system.  
-You can find the latest version on the [Releases](https://github.com/Antosik/lol-guilds-client/releases) page.
+## Как использовать
 
-## Contributing
-- Submit bug reports and feature requests
-- Review source code  
+1. Скачайте и установите последнюю версию приложения ([Windows](https://github.com/Antosik/lol-guilds-client/releases/download/v0.0.5/League-Guilds-Client-Setup-0.0.5.exe), [Linux](https://github.com/Antosik/lol-guilds-client/releases/download/v0.0.5/League-Guilds-Client-0.0.5.AppImage), [macOS](https://github.com/Antosik/lol-guilds-client/releases/download/v0.0.5/League-Guilds-Client-0.0.5.dmg))
```
Comment (kozhin) [inline README.md]: Может быть, стоит сделать какую-то постоянную ссылку на самую последнюю версию? Например, на страничку релизов на GitHub.

File: README.md (hunk 1 above)
Comment (Antosik) [reply README.md]: Исправил с:

File: README.md (hunk 2 above)
Comment (Antosik) [reply README.md]: Исправил 👌

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
-- Operating system: [e.g. Windows 10]  
-- Application Version [e.g. 0.0.5]  
+**Как ее получить / To Reproduce**  
+Какие шаги вы прошли, чтобы ее получить / Steps to reproduce the behavior.
 
-**Additional context**  
-Add any other context about the problem here.  
\ No newline at end of file
+**Скриншоты / Screenshots**  
+Если возможно, прикрепите скриншот с ошибкой / If applicable, add screenshots to help explain your problem.
```
Comment (kozhin) [inline .github/ISSUE_TEMPLATE/bug_report.md]: add screenshots to help explain your problem.
->
 add screenshots to provide more details about your problem.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -1,19 +1,20 @@
...
+Какие шаги вы прошли, чтобы ее получить / Steps to reproduce the behavior.
 
-**Additional context**  
-Add any other context about the problem here.  
\ No newline at end of file
+**Скриншоты / Screenshots**  
+Если возможно, прикрепите скриншот с ошибкой / If applicable, add screenshots to help explain your problem.  
+
+**Среда выполнения / Environment:**  
+Опишите, в какой ОС и на какой версии приложения возникает ошибка / Describe in which OS and on which version of the application the error occurs.
```
Comment (kozhin) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Describe in which OS and on which version of the application the error occurs.
->
Describe which OS and version your are using as well as the version of the League Guilds Client

**Pull request merged**